### ISSUE TITLE
Compatibility with visibility

### ISSUE THREAD

**Issue by lrobidou** [snowfoxsh/nestify#1]
Thank you for your crate! One thing that prevents me from using it is the fact that it doesn't seem to handle visibility (or that I don't know how to do it!).
```rust
nest! {
    pub enum Level0 {
        Level1(enum Level1 {
                Level2(enum Level2 {
                    A,
                    B,
                }),
        }),
    }
}
```
Level0 will be public, but not Level1 and Level2. How can I make Level1 and Level2 public as well?

**Comment by snowfoxsh** [comment]
I will work on finding a solution to that!

**Comment by snowfoxsh** [comment]
Issue should be fixed per #3. This issue will be closed within 48 hours
